backend/migrations/20251030239022-alter-note-text-to-string.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.changeColumn("SeasonChangeLogs", "notes", {
+      type: Sequelize.STRING(2000),
```

Comment:
You can actually leave this as TEXT.  

TEXT is a varialbe length string that can hold up to 1GB of data

Reply:
As we had a chat in the meeting, it has a different type `character varying(255)` in the DB, so I needed to change it manually.